xclim/core/calendar.py

```diff
@@ -486,7 +378,7 @@ def percentile_doy(
         # Preserve chunk size
         time_chunks_count = len(arr.chunks[arr.get_axis_num("time")])
         doy_chunk_size = np.ceil(len(rrr.dayofyear) / (window * time_chunks_count))
-        rrr = rrr.chunk(dict(stack_dim=-1, dayofyear=doy_chunk_size))
+        rrr = rrr.chunk({"stack_dim": -1, "dayofyear": doy_chunk_size})
 
```

Comment:
using {} instead of dict: is that a personal stylistic choice, or it's best practices and some pre-commit check complains about it? Just so I know for the future

Reply:
Best practices, apparently. Using `dict()` is good when you want to convert an object to a dict and relies on a C-lang internal, while using curly brackets seems to be slightly optimized for Python.

More context: https://stackoverflow.com/questions/664118/whats-the-difference-between-dict-and